package.json

```diff
@@ -14,22 +11,23 @@
   "scripts": {
     "add-dev": "node ./scripts/add-dev-dependency.js",
     "sync-dev": "node ./scripts/sync-dev-dependency.js",
-    "build": "yarn workspace ember-data build",
-    "build:production": "yarn workspace ember-data build:production",
+    "build": "pnpm --filter ember-data build",
+    "build:production": "pnpm --filter ember-data build:production",
     "lint:features": "node ./scripts/lint-features.js",
     "lint:js": "eslint --cache --ext=js,ts .",
     "lint:js:changed": "node ./scripts/lint-changed-js.js",
+    "preinstall": "npx only-allow pnpm",
     "problems": "tsc -p tsconfig.json --noEmit --pretty false",
-    "start": "yarn workspace ember-data start",
+    "start": "pnpm --filter ember-data start",
     "test": "lerna run --stream --no-private --ignore='@ember-data/{private-build-infra,canary-features,debug}' test",
-    "test:all": "yarn workspace ember-data test:all",
+    "test:all": "pnpm --filter ember-data test:all",
```

Comment:
my suspicion is that --filter won't work in a lot of these script scenarios the way we need it to, because it will match anything with an `@ember-data/<foo>` name too :/